before-commit.sh

```diff
@@ -51,6 +52,14 @@ if [ ${goBuildResult} != 0 ]; then
 else echo -e "${GREEN}√ go build ${binary} ${NC}"
 fi
 
+${buildEnv} go build -o health-proxy ./cmd/healthcheck/main.go
+goBuildResult=$?
+if [ ${goBuildResult} != 0 ]; then
+    echo -e "${RED}✗ go build ${binary} ${NC}\n $goBuildResult${NC}"
```

Comment:
`${binary}` ?


Also is that step necessary as we are using multistage builds for healthcheck?